CMakeLists.txt

```diff
@@ -48,21 +50,40 @@ find_package(ZLIB REQUIRED)
 find_package(Threads REQUIRED)
 
 # Clang
-include(cmake/clang/download.cmake)
+if (${DOWNLOAD_CLANG})
+  set(DOWNLOAD_CLANG {$DOWNLOAD_CLANG})
+endif()
```

Comment:
I don't understand the meaning of these three lines. What's the point in doing this?